interfaces/powermeter.yaml

```diff
@@ -26,3 +26,6 @@ vars:
     description: Measured dataset
     type: object
     $ref: /powermeter#/Powermeter
+  public_key:
```

Comment:
@corneliusclaussen @hikinggrass @Pietfried Can we get an opinion from Pionix on this change? It seems straight forward enough for us, but just wanted to hear from you.

Reply:
We can revert this since the public key is now part of the signed meter message